pyof/foundation/base.py

```diff
@@ -386,7 +386,7 @@ def replace_pyof_version(module_fullname, version):
 
     @staticmethod
     def get_pyof_obj_new_version(name, obj, new_version):
-        """Return a class atrribute on a different pyof version.
+        r"""Return a class attribute on a different pyof version.
```

Comment:
@diraol, in this case, is necessary. Below we have a binary number b'\x01', and the symbol '\' is used.